RInAppMessaging/Classes/Utilities/BundleInfo.swift

```diff
@@ -24,15 +24,19 @@ internal class BundleInfo {
 internal extension Bundle {
 
     static var sdk: Bundle? {
-        return Bundle(identifier: "org.cocoapods.RInAppMessaging")
+        if let defaultBundle = self.init(identifier: "org.cocoapods.RInAppMessaging") {
+            return defaultBundle
+        } else {
+            return (allBundles + allFrameworks).first(where: { $0.bundleIdentifier?.contains("RInAppMessaging") == true })
+        }
```

Comment:
Maybe
```
let defaultBundle = self.init(identifier: "org.cocoapods.RInAppMessaging")
return defaultBundle ?: (allBundles + allFrameworks).first(where: { $0.bundleIdentifier?.contains("RInAppMessaging") == true })
```
is nicer?

And the bundle matching could even be its own tiny extension to improve readability?

Reply:
Yeah, sounds better. And also I forgot to add assertion.